gradio.egg-info/PKG-INFO

```diff
@@ -1,11 +1,14 @@
-Metadata-Version: 1.0
+Metadata-Version: 2.1
 Name: gradio
 Version: 2.7.0b70
 Summary: Python library for easily interacting with trained machine learning models
 Home-page: https://github.com/gradio-app/gradio-UI
 Author: Abubakar Abid, Ali Abid, Ali Abdalla, Dawood Khan, Ahsen Khaliq, Pete Allen, Ömer Faruk Özdemir
 Author-email: team@gradio.app
 License: Apache License 2.0
-Description: UNKNOWN
 Keywords: machine learning,visualization,reproducibility
 Platform: UNKNOWN
+License-File: LICENSE
+
+UNKNOWN
```

Comment:
Why UNKNOWN on a line by itself? That seems off

Reply:
I don't have any idea, it just updated after local gradio installation, shall I remove it?

Reply:
I think so yeah

Reply:
I think so yeah

Reply:
I think so yeah